membership_product_set/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2019 Yu Weng <yweng@elegosoft.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Product Set for memberships',
+    'version': '11.0.1.0.0',
+    'license': 'AGPL-3',
+    'category': 'Association',
+    'author': 'Yu Weng, '
+              'Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/vertical-association',
+    'depends': [
+        'membership',
```

Comment:
Where is that module 'membership' defined ? I don't see it in the v11 branch.

Reply:
It is here.
https://github.com/odoo/odoo/tree/11.0/addons/membership